kafka.go

```diff
@@ -50,34 +51,47 @@ type KafkaMessage struct {
 	ReqHeaders map[string]string `json:"Req_Headers,omitempty"`
 }
 
+// ErrorTLSMissingKey is the error when client cert is present but key is missing
+var ErrorTLSMissingKey = errors.New("Missing key of client certificate")
+// ErrorTLSMissingCert is the error when key is present but client cert is missing
+var ErrorTLSMissingCert = errors.New("Missing client certificate")
+
 // NewTLSConfig loads TLS certificates
 func NewTLSConfig(clientCertFile, clientKeyFile, caCertFile string) (*tls.Config, error) {
 	tlsConfig := tls.Config{}
 
+	if clientCertFile != "" && clientKeyFile == "" {
+		return &tlsConfig, ErrorTLSMissingKey
```

Comment:
making them more descriptive

```suggestion
		return &tlsConfig, errors.New(mMissing key of client certificate in kafka")
```